### DISCUSSION TITLE
Option to adjust the chance of cauldrons being filled by dripstone water or lava.

### DISCUSSION THREAD

**Discussion by WanderingLemon** [PurpurMC/Purpur#596] [__Completed__] [answered]
Option to adjust the chance of cauldrons being filled by dripstone water or lava. It would be nice to have this since we already have the ability to adjust the chance of cauldrons filling with powdered snow or rain.

**Comment by encode42** [comment]
This is entirely affected by randomtickspeed iirc, not much we can do. I'll look it over again in case there's anything I missed

**Comment by encode42** [comment, marked as answer]
Was implemented in ad2bac1e243835f9eb09596bd1453e76e8845a0c! [Config options](https://purpurmc.org/docs/Configuration/#fill-chances)
